pennylane/qnode_new/device_jacobian.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Device Jacobian QNode.
+
+A QNode that delegates all gradient computations directly to the device.
+"""
+from pennylane.qnode_new.jacobian import JacobianQNode
+
+
+class DeviceJacobianQNode(JacobianQNode):
+    """Quantum node that delegates gradient computation to the device"""
+
+    # pylint: disable=abstract-method
+
+    def _best_method(self, idx):
+        """Determine the correct partial derivative computation method for a free parameter.

```

Comment:
I would not use the term "free parameter" anywhere if possible. In the JacobianQNode PR I tried to change every occurrence of "free parameter" into something more specific, usually "positional parameter".

Ultimately we need to decide how the nomeclature goes and then do a search and replace, in JacobianQNode I consistently used the placeholder names "positional parameter" (differentiable) and "auxiliary parameter" (nondifferentiable).

Reply:
```suggestion
        """Determine the correct partial derivative computation method for a positional parameter.
```